src/index.html

```diff
@@ -1,13 +1,291 @@
+                </div>
+
+                <h3 class="category__title">Accessories</h3>
+              </article>
+
+            </div>
+            <div class="buttons">
+              <button type="button" class="buttons__config category__button--wide">
+                View all
+              </button>
+            </div>
+
+          </section>
+
+        </div>
+      </div>
+
+      <div class="full-width-image">
+        <img src="./images/full-width-image.png"
+          alt="photo" class="full-width-image__content">
+      </div>
+
+      <div class="container">
+        <div class="page__main__content">
+          <section class="about-us">
+            <h2 class="section-title about-us__title">About us</h2>
+
+            <p class="about-us__text">
+                We’re a London-based tech company on a mission to remove barriers between people and technology.
+              <br><br>
+                To do this, we’re crafting intuitive, flawlessly connected products that
+                improve our lives without getting in the way.
+                No confusing tech-speak. No silly product names. Just artistry, passion and trust.
+                And products we’re proud to share with our friends and family. Simple.
+              <br><br>
+                Because beautiful tech shouldn’t be complicated.
+            </p>
+          </section>
+
+          <section class="contacts" id="contacts">
+            <h2 class="section-title contacts__form">Contact us</h2>
+
+            <form class="contacts__form" onsubmit="this.reset(); return false;">
+              <input
+                class="contacts__content"
+                type="text"
+                id="name"
+                name="user_name"
+                placeholder="Name"
+                required
+              >
+
+              <input
+                class="contacts__content"
+                type="email"
+                id="email"
+                name="user_name"
+                placeholder="Email"
+                required
+              >
+
+              <textarea
+                name="message"
+                cols="30"
+                rows="10"
+                placeholder="Message"
+                class="contacts__content"
+                required
+               ></textarea>
+
+              <div class="buttons">
+                <button type="submit" class="buttons__config contacts__button">
+                  Send
+                </button>
+              </div>
+            </form>
+
+            <address class="contacts__address">
+              <div class="contacts__block">
+                <p class="contacts__subtitle">Phone</p>
+                <a href="tel:+1 234 5555-55-55" class="contacts__link link">+1 234 5555-55-55</a>
+              </div>
+              <div class="contacts__block">
+                <p class="contacts__subtitle">Email</p>
+                <a href="mailto:hello@nothing.com" class="contacts__link link">hello@miami.com</a>
```

Comment:
Change the email address to 'hello@nothing.com' instead of 'hello@miami.com' to match the company name.